TrueCraft.API/Ray.cs

```diff
@@ -5,12 +5,24 @@
 
 namespace TrueCraft.API
 {
-    // Mostly taken from the MonoXna project, which is licensed under the MIT license
```

Comment:
Okay, I'll remove the remark tag around them so they aren't visible in Intellisense.